### PR TITLE
Some refactoring of pages + changes to text and typos

### PR DESCRIPTION
**Pull request by jeffgeudens**
Based on https://github.com/botanix-labs/botanix/issues/626

### CODE REVIEW

File: book/intro.md [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
-#### Alpha software warning copy
-Botanix Federation is a alpha software that has not been audited. Please only deposit funds that you are willing to lose.
+<div class="warning">
 
-**Note**: this book is copied over from The Reth book modified to fit the requirments needed for the Botanix federation.
+**Alpha software warning copy**
+
+Warning blocks should be used sparingly in documentation, to avoid "warning
+fatigue," where people are trained to ignore them because they usually don't
+matter for what they're doing.
```
Comment (arminsabouri) [inline book/intro.md]: Not sure I understand this

File: book/run/observability.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Observability with Prometheus & Grafana
+# Logs and observability
```
Comment (arminsabouri) [inline book/run/observability.md:1]: Why truncate this?

File: book/cli/poa.md [hunk 3]
```diff
@@ -144,7 +144,7 @@ Networking:
           Maximum number of inbound requests. default: 30
 
       --pooled-tx-response-soft-limit <BYTES>
-          Soft limit for the byte size of a `PooledTransactions` response on assembling a `GetPooledTransactions` request. Spec'd at 2 MiB.
+          Soft limit for the byte size of a `PooledTransactions` response on assembling a `GetPooledTransactions` request. Spec`d at 2 MiB.
```
Comment (arminsabouri) [inline book/cli/poa.md:147]: why a back tick here?

File: book/cli/poa.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# `reth poa`
```
Comment (arminsabouri) [inline book/cli/poa.md]: technically this should be `reth poa` as we are running the reth binary with the poa command.

I think this is fine

**Review by arminsabouri — CHANGES_REQUESTED** [review]

File: book/intro.md (hunk 1 above)
Comment (jeffgeudens) [reply book/intro.md]: Sorry, this was a mistake. Forgot to replace the text. Will fix.

File: book/run/observability.md (hunk 2 above)
Comment (jeffgeudens) [reply book/run/observability.md:1]: To have consistency on all pages. Same wording in the title on the page, table of contents on the left and list overview on the "Run a node" page.

File: book/cli/poa.md (hunk 3 above)
Comment (jeffgeudens) [reply book/cli/poa.md:147]: Makes that the formatting looks better on the page. If you visit https://books.botanixlabs.dev/cli/poa.html , you'll see that a lot of the code is yellow. This is solved by using the back tick here.

File: book/cli/poa.md (hunk 4 above)
Comment (jeffgeudens) [reply book/cli/poa.md]: Will change it back

**Review by arminsabouri — APPROVED** [review]
Commits squashed. 
lets wait till CI passes then good to merge

**Pull request merged**